did-git-spec.md

```diff
@@ -54,13 +58,13 @@ The commit ID portion becomes a globally unique identifier for the Git repositor
 
 # CRUD Operations
 
-## Create 
+## Create
 
 **1. Repository DID**
 
 The operation of creating a git-did can be done at any point in a git repository.
 
-A DID Document should be created at `.did/repo.did` and all maintainer DID documents added into the .did/ directory. All files are then signed and committed by one of the associated keys in the authentication block into the project.  The SHA1 of the commit is then used to create the git-did for this respository. 
+A DID Document should be created at `.did/repo.did` and all maintainer DID documents added into the .did/ directory. All files are then signed and committed by one of the associated keys in the authentication block into the project.  The SHA1 of the commit is then used to create the git-did for this respository.
```

Comment:
I think we decided that we're not going to have a hidden '.did' directory. The directory is just 'did' in the root of the repo.

Reply:
should be fixed by the rebase